interp/handler_test.go

```diff
@@ -67,40 +67,40 @@ var modCases = []struct {
 	want    string
 }{
 	{
-		name: "ExecBlacklist",
-		exec: blacklistBuiltinExec("sleep"),
+		name: "Execblocklist",
+		exec: blocklistBuiltinExec("sleep"),
 		src:  "echo foo; sleep 1",
-		want: "foo\nsleep: blacklisted builtin",
+		want: "foo\nsleep: blocklisted builtin",
 	},
 	{
 		name: "ExecWhitelist",
```

Comment:
"blacklist" => "blocklist", "whitelist" => ?

Reply:
forgot this one. it's not even an allowlist.